README.md

```diff
@@ -100,3 +100,6 @@ ds = RayMLDataset.from_spark(..., fs_directory="hdfs://host:port/your/directory"
 
 ## More Examples
 Not sure how to use RayDP? Check the `examples` folder. We have added many examples showing how RayDP works together with PyTorch, TensorFlow, XGBoost, Horovod, and so on. If you still cannot find what you want, feel free to post an issue to ask us!
+
+## Spark Submit
+RayDP provides a substitute for spark-submit in Apache Spark. You can run your java or scala application on RayDP cluster by using `bin/raydp-submit`. You can add it to `PATH` for convenience. When using `raydp-submit`, you should specify number of executors and number of cores each executor by `--conf spark.executor.cores=1` and `--conf spark.executor.instances=1`. `raydp-submit` only supports Ray cluster. Spark standalone, Apache Mesos, Apache Yarn are not supported, please use traditional `spark-submit` in that case. For the same reason, you do not need to specify `--master` in the command.
```

Comment:
We should mention `spark.executor.memory` as well, right? And, this should be the Spark client mode, we maybe should mention it too.